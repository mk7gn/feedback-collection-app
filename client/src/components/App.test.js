import React from 'react';
import { render, cleanup } from '@testing-library/react';

import App from './App';

afterEach(cleanup);

test('should render component', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});