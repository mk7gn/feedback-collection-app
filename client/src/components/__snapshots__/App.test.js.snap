// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div>
      <nav>
        <div
          class="nav-wrapper"
        >
          <a
            class="brand-logo"
            href="/"
          >
            Emaily
          </a>
          <ul
            class="right hide-on-med-and-down"
            id="nav-mobile"
          >
            <li>
              <a
                href="/auth/google"
              >
                Login with Google
              </a>
            </li>
          </ul>
        </div>
      </nav>
      <h2>
        Landing
      </h2>
    </div>
  </div>
</DocumentFragment>
`;
